src/ontology/mondo-edit.obo

```diff
@@ -585022,6 +585026,21 @@ is_a: MONDO:0004981 {source="PMID:38033089"} ! atrial fibrillation
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-3458-4839
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7886" xsd:anyURI
 
+[Term]
+id: MONDO:1030015
+name: acquired porphyria
+def: "A porphyria disorder that is acquired during the lifetime of the individual." [MONDO:patterns/acquired]
+xref: GARD:10353 {source="MONDO:GARD"}
+xref: ICD9:277.1 {source="MONDO:equivalentTo"}
+xref: MESH:D011164 {source="MONDO:relatedTo"}
+xref: NCIT:C97096 {source="MONDO:relatedTo"}
+xref: SCTID:154768004 {source="MONDO:equivalentTo"}
+xref: SCTID:86292002 {source="MONDO:equivalentTo"}
+is_a: MONDO:0037939 ! porphyria
+intersection_of: MONDO:0037939 ! porphyria
```

Comment:
This SC annotation needs a source

Reply:
added

Reply:
GARD:20353 is not an exact match, so we should not use it as a source for this SC. 
Is there a PMID where they mention that there are acquired forms of porphyria? Maybe a review?